mne_neuron/dipole.py

```diff
@@ -97,15 +102,14 @@ def simulate_dipole(net, trial=0, inc_evinput=0.0, verbose=True, extdata=None):
     from neuron import h
     h.load_file("stdrun.hoc")
 
-    # maintain vectors across trials
-    global t_vec, dp_rec_L2, dp_rec_L5
+    t_vec, dp_rec_L2, dp_rec_L5 = initialize_sim(net)
+
+    # make sure network state is consistent
+    net.state_init()
 
-    if trial == 0:
-        initialize_sim_once(net)
-    else:
-        net.state_init()
-        # adjusts the rng seeds and then the feed/event input times
-        net.reset_src_event_times(inc_evinput = inc_evinput * (trial + 1))
+    if trial != 0:
+        # for reproducibility of original HNN results
+        net.reset_src_event_times()
```

Comment:
Note that the inc_evinput parameter is no longer necessary to reproduce the same results as the original HNN, but reset_src_event_times() cannot be called on the 1st trial. The number of times that reset_src_event_times() is the sole difference between trials.